sofiadinizms/vida-de-inseto
# feat(feature/level-2): created LevelTwoView with pinch gesture mechanics

### What I did

Created the Level Two general mechanics: use the pinch gesture to change mushroom's size
Added temporary assets to demonstrate said mechanics

### How to test

Git pull this branch
Run it on an Xcode simulator
Check if a mushroom appear on the center of the screen
Use option key while clicking the mouse in the area you wish to pinch. This brings up a pair of dots that represent your fingertips.

If you managed to successfully complete all the task described above, then everything is working correctly
